content/efcore/getting-started/samples/loading-related-data-tutorial/load-method/Program.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class Program
+{
+    public static void Main()
+    {
+        using (var context = new LibraryContext())
+        {
+            var author = context.Authors
+                .Single(a => a.LastName == "Twain");
+
+            context.Entry(author)
+                .Collection(a => a.Books)
+                .Query()
+                .Where(b => b.Title.Contains("Huck"))
+                .Load();
+            Console.WriteLine(book.Title);
```

Comment:
`book` doesn't exist yet...we could just do a foreach loop to show that it is the only book of his that was loaded, is there an example where we show it wasn't initially loaded? Cuz we could do that before loading, but it's also fine the way it is
 
```
foreach (Book book in author.Books)
 {
    Console.WriteLine(book.Title);
}
```